OBF Bylaws.md

```diff
@@ -67,26 +67,25 @@ ARTICLE III - BOARD OF DIRECTORS
         (2)	Account for all monies received by the OBF and its committees
         (3)	Provide an annual balance sheet for the OBF and keep the Board informed as to the fiscal status of OBF.
         (4)	Prepare an annual budget
-    d.	Parliamentarian
-        (1)	Interpret and rule on procedural matters at all meetings.
-        (2)	Chair the Nominating committee.
-        (3)	Chair the Bylaws Review committee.
-    e.	Board Member(s) at Large
+        (5)	Serve in the role of Project Liaison, with the rights and obligations conveyed by this role, to Software in the Public Interest, Inc. (SPI), while OBF is an associated project of SPI.
```

Comment:
This is clear on the expanded diff view on GitHub - we are proposing the SPI Project Liaison role be switched from the OBF President to the OBF Treasurer.